.gitignore

```diff
@@ -1,4 +0,0 @@
-.cache
```

Comment:
Why is the gitignore files deleted?

Reply:
I didn't think u needed my git ignore file.  Uploaded it by mistake.

Reply:
Thats the gitignore file that we use, ummm didn't think you uploaded it.

Reply:
i deleted the request to merge my gitignore with your gitignore